google/cloud/pubsublite/internal/stream_factory.h

```diff
@@ -0,0 +1,85 @@
+#include "absl/container/flat_hash_map.h"
+#include <google/cloud/pubsublite/v1/publisher.grpc.pb.h>
+
+namespace google {
+namespace cloud {
+namespace pubsublite_internal {
+
+template <class Request, class Response>
+using BidiStream = google::cloud::internal::AsyncStreamingReadWriteRpc<Request, Response>;
+
+template <class Request, class Response>
+using StreamFactory = std::function<std::unique_ptr<BidiStream<Request, Response>>()>;
+
+using ClientMetadata = absl::flat_hash_map<std::string, std::string>;
+
+namespace stream_factory_internal {
+
+inline std::unique_ptr<grpc::ClientContext> MakeGrpcClientContext(const ClientMetadata& metadata) {
+    auto context = std::make_unique<grpc::ClientContext>();
```

Comment:
s/std::make_unique/absl::make_unique/ the library needs to support C++11

Reply:
Done.